cthulhu/cthulhu/manager/eventer.py

```diff
@@ -316,6 +316,11 @@ def _on_osd_map(self, fsid, new, old):
         created_osds = new_osd_ids - old_osd_ids
 
         def osd_event(severity, msg, osd_id):
+            osd_uuid = ''
+            for osd in old.data['osds'] + new.data['osds']:
```

Comment:
This change makes all calls to osd_event O(N) where previously it was O(1). Is there a better way to get osd uuid ?